### ISSUE TITLE
Port BladeView

### ISSUE THREAD

**Issue by niels9001** [CommunityToolkit/Windows#89]
```[tasklist]
### Base Porting Checklist
- [ ] Component name (to `CommunityToolkit.WinUI.Controls.BladeView`)
- [ ] Redesign, following the latest WinUI design guidelines
- [ ] Merge differences from UWP main to WinAppSDK / Uno forks
- [ ] Support for WASM via Uno Platform
- [ ] Improve Samples
- [ ] All brought over tests pass for UWP/WinAppSDK
```

```[tasklist]
### Known Issues
```

```[tasklist]
### Improvements
```

## Notable Changes
N/A

**Comment by zhuxb711** [comment]
Hope BladeView could be merge into the new CommunityToolkit

**Comment by 0x5bfa** [comment]
I'd appreciate if this is a priority work.

**Comment by niels9001** [comment]
@zhuxb711 @0x5bfa Is this something you'd like to pick up and work on?

**Comment by 0x5bfa** [comment]
Yesn't. We have other things (EnableMiddleClickScrolling and SystemInformation that block us from enabling AoT compilation. If nothing out of these will not have ported, I'd work on in a few months.

**Comment by michael-hawker** [comment]
There was an older discussion here related to both BladeView and ListDetailsView here: https://github.com/CommunityToolkit/Labs-Windows/discussions/280

**Comment by 0x5bfa** [comment]
We began to migrate to WCT 8.x but had to port a lot fyi (`https://github.com/files-community/Files/pull/16741`)

**Comment by zhuxb711** [comment]
@0x5bfa Will you contribute the BladeView to 8.x CommunityToolkit in recent days?

**Comment by 0x5bfa** [comment]
It went well, I will do that for real quick if you'd like. CC @Arlodotexe @niels9001

**Comment by niels9001** [comment]
> It went well, I will do that for real quick if you'd like. CC [@Arlodotexe](https://github.com/Arlodotexe) [@niels9001](https://github.com/niels9001)

Sure, go ahead!

**Comment by 0x5bfa** [comment]
FYI was told that this is blocked by the migration assessment in https://github.com/CommunityToolkit/Windows/issues/602

**Comment by zhuxb711** [comment]
Did anyone plan to continue the porting job on BladeView? @0x5bfa Could you reopen your PR to finish the porting? @Arlodotexe Why the migration assessment will block the porting process even @0x5bfa have opened an actual PR for that?

**Comment by Arlodotexe** [comment]
- Porting anything from 7x to 8x requires rigorous evaluation across source, samples and tests per our [component quality evaluation criteria](https://github.com/CommunityToolkit/Windows/wiki/Component-Quality-Evaluation).
- For many un-ported 7x components, source/samples/tests don't meet [modern quality expectations](https://github.com/CommunityToolkit/Windows/wiki/Component-Quality-Evaluation#when-expectations-arent-met).
- Since quality flows from maintainability through developer experience to user experience (MX → DX → UX), gaps earlier in this chain affect everything downstream:
    - Poor maintainability makes it hard to improve the API
    - Poor API design makes it hard to create good user experiences
- We avoid parallel APIs if possible through proactive API design
- Breaking removals only happen in major version bumps (e.g. code removed between 7x->8x)
- We do not make major breaking changes to already stable-released APIs (e.g. can't just port the 7x code as-is then break it later)
- If too complex → component may return to Labs for redesign (common for refactors, uncommon for e.g. style updates). 

tl;dr; Before 8x, it was common for the original component authors to build in semi-isolation then disappear once their code shipped, never to follow-up again. Most old 7x code simply wasn't design or maintained well enough, so everything needs to be reviewed and evaluated in terms of design (visual/api), extensibility, usability, maintainability, etc. across source, samples/docs, and tests.

**Comment by michael-hawker** [comment]
I had linked to an orthogonal discussion about a similar but either new or composited experience here above: https://github.com/CommunityToolkit/Labs-Windows/discussions/280

I knew we had more about a modern BladeView design, which I finally found in the old repo here: https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/4701

And then here are a couple of old bugs from the old control around naming and behavior issues:
- https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/3597
- https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/4700

If anyone wants to take on a plan for modernizing BladeView (and or ListDetailsView), please let us know. That work can initially happen in WCT Labs in order to lower the barrier to entry and experiment with one or more approaches.

**Comment by zhuxb711** [comment]
No BladeView is one of the main blocker preventing me from using WCT 8.x. 
### "No one using BladeView so the priority is low"
> "They just simply don't speak aloud and stays in WCT 7.1.2"
